lib/rules/require-capitalized-comments.js

```diff
@@ -52,7 +52,7 @@
  * ```
  *
  * ```js
- * "requireCapitalizedComments": { allExcept: ["jshint"] }
+ * "requireCapitalizedComments": { "allExcept": ["jshint"] }
```

Comment:
Because without it an error is thrown. The property needs to be quoted.
On Apr 29, 2015 9:57 PM, "Eli White" <notifications@github.com> wrote:

> In lib/rules/require-capitalized-comments.js
> <https://github.com/jscs-dev/node-jscs/pull/1322#discussion_r29368414>:
>
> > @@ -52,7 +52,7 @@
> >   * ```
> >   *
> >   * ```js
> > - * "requireCapitalizedComments": { allExcept: ["jshint"] }
> > + * "requireCapitalizedComments": { "allExcept": ["jshint"] }
>
> Why is this necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jscs-dev/node-jscs/pull/1322/files#r29368414>.
>